### PR TITLE
[14.0][ADD]stock_package_autoload: new module

### PR DESCRIPTION
**Pull request by PicchiSeba**
This module allows to automatically add the content of a package to the transfer lines

### CODE REVIEW

**Comment by PicchiSeba** [comment]
Please let me know if this could be an interesting addition to the repo and if so, if tests are needed

File: stock_package_autoload/models/stock_move.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        )
+        return to_return
+
+    def _compute_package_domain(self):
+        """
+        There's no need to compute this field if the current user doesn't have the
+        necessary group
+        """
+        if not self.env.user.has_group("stock.group_tracking_lot"):
+            self.write({"package_domain": "[]"})
```
Comment (HekkiMelody) [inline stock_package_autoload/models/stock_move.py:43]: **issue**: missing a `return` here.

**Comment by PicchiSeba** [comment]
I added some tests

**Review by HekkiMelody — COMMENTED** [review]
Just a minor fix, otherwise LGTM

**Review by HekkiMelody — APPROVED** [review]
Code review, LGTM

**Pull request closed without merging**